src/ui/server/helpers/required-fields/eligibility/index.ts

```diff
@@ -12,14 +12,15 @@ const {
     HAS_END_BUYER,
     TOTAL_CONTRACT_VALUE,
     TOTAL_CONTRACT_VALUE_ID,
+    HAS_REVIEWED_ELIGIBILITY,
     ...FIELD_IDS
   },
 } = INSURANCE_FIELD_IDS;
 
 /**
  * List of field IDs that are not relevant.
  */
-export const irrelevantFields = [BUYER_COUNTRY_ISO_CODE, COVER_PERIOD_ID, HAVE_AN_ACCOUNT, TOTAL_CONTRACT_VALUE_ID];
+export const irrelevantFields = [BUYER_COUNTRY_ISO_CODE, COVER_PERIOD_ID, HAVE_AN_ACCOUNT, TOTAL_CONTRACT_VALUE_ID, HAS_REVIEWED_ELIGIBILITY];
```

Comment:
`HAS_REVIEWED_ELIGIBILITY` is relevant 😛  Should not be able to navigate to this page without all other eligibility questions answers provided

Reply:
Will be removed once I add changes to create application after the next PR

Reply:
It needs to be there, this is not related to application creation?